cmd/pomerium/main.go

```diff
@@ -213,11 +213,15 @@ func parseOptions(configFile string) (*config.Options, error) {
 	metrics.AddPolicyCountCallback(o.Services, func() int64 {
 		return int64(len(o.Policies))
 	})
-	checksumInt, err := strconv.ParseInt(fmt.Sprintf("0x%s", o.Checksum()), 0, 64)
+
+	float := big.Float{}
```

Comment:
🚨🚨🚨Super nit-picky alert 🚨🚨🚨 

I like using the form `var float big.Float` in cases like this because I think it makes it more clear we are [not initializing the variable](https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_declaration_style).  I'll defer to your preference. 😄

Reply:
Ah yes that's fine.  The usage of Float is weird in general; I updated to something a little more compact after a second glance.